extensions/downloads-manager/src/open-downloads.tsx

```diff
@@ -0,0 +1,9 @@
+import { open, popToRoot, showHUD } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+
+const openFinderToDownloadsAppleScript = `do shell script "open ~/Downloads/"`;
+
+export default async function main() {
+  await runAppleScript(openFinderToDownloadsAppleScript);
```

Comment:
No need for AppleScript in this case. just use our `open` or `showInFinder` function.

```suggestion
  const downloadsdir = join(homedir, "Downloads");
  await showInFinder(downloadss);
```

You need to fix the imports ;-)

Reply:
Cool!  I will switch to using that approach.